src/MailboxInterface.php

```diff
@@ -105,9 +105,9 @@ public function getIterator(): MessageIteratorInterface;
      *
      * @param string $message
      *
-     * @return bool
+     * @return int
      */
-    public function addMessage(string $message): bool;
+    public function addMessage(string $message): int;
```

Comment:
Note that a signature change would require a new major version bump.
Major version bumps are done once every six months.